en/api/action.md

```diff
@@ -41,7 +41,7 @@ Chart actions supported by ECharts are triggered through [dispatchAction](~echar
 Highlights specified data graphics.
 
 ```ts
-// If highlight series:
+// If highlight series（in the `sunburst` series,`highlight` has been deprecated since `v5.0.0`, please use `sunburstHighlight` instead）：

```

Comment:
I think `sunburstHighlight` is deprecated rather than `highlight` and your current change conflicts with your previous change in apache/echarts#17587.

Reply:
Also, the changes don't match your PR title. I didn't see any fix for spelling errors.

Reply:
I have modified it  a moments ago,please view again, thanks